### PR TITLE
increase pkg/net/cidrs unit test coverage to 100%

### PR DESCRIPTION
**Pull request by BenTheElder**
just to be on the safe side. coverage was good before at 94% for triemap.go and 100% for the rest.

### CODE REVIEW

File: pkg/net/cidrs/triemap.go [hunk 1]
```diff
@@ -272,21 +271,20 @@ func (t *trieMap) getIPv6(addr netip.Addr) (int, bool) {
...
+				break
 			}
 		}
 		// check for a match in the current node
 		if curr.value != nil && curr.value.cidr.Contains(addr) {
 			return curr.value.key, true
 		}
 	}
-	// we walked to the final IP bit without a match
 	return -1, false
```
Comment (BenTheElder) [inline pkg/net/cidrs/triemap.go:289]: these were unreachable / dead code before, consolidated with the dead-end returns => `break` which we always hit. I would hope it compiles the same ...

Comment (hh) [reply pkg/net/cidrs/triemap.go:289]: 👍

**Review by hh — APPROVED** [review]

**Pull request merged**